### ISSUE TITLE
I can not copy text from network manager page in Dev-Tools.

### ISSUE THREAD

**Issue by buraksafak** [flutter/devtools#5031]
<-- Please describe your problem here. Be sure to include repro steps. -->
___
**DevTools version**: 2.15.0
**Connected Device**:
CPU / OS: x64 (64 bit) android
Dart Version: 2.18.6
Flutter Version: 3.3.10 / stable
Framework / Engine: 135454af32 / 3316dd8728
Connected app type: Flutter native (debug build)

**Comment by kenzieschmoll** [comment]
Specifically, what text on the network page are you trying to copy but cannot?

**Comment by julemand101** [comment]
@kenzieschmoll 
Tested with DevTools 2.20.1 in Firefox and Edge, and I would say in general the behavior of copying text from the Network GUI are quite bad. I have tested with 

- Only field I can copy is the Uri column and only for a single column at a time.
    - I would here expect a way to copy multiple rows from the table at the same time as text like a normal HTML table works.
    - I also expected to copy all details from the table to e.g. attach the details to issues.
- When selecting a network event, I cannot copy the text in "Overview" as I would expect (selecting all text and copy). Instead, I can either select a single description or a single value.
    - E.g. I can select the text "Socket id:  " and then select "2641797649968". But I cannot make a combined selection that contains both like "Socket id:  2641797649968".
    - This makes it near useless to use DevTools to collect debug information and attach this as text.

These issues seems to be consistent with tables in DevTools in general. E.g. the same problematic behavior can be found in the "VM Tools" tab where all tables prevent you from copying text from them. E.g. an example would be the "Instructions" view where we can see the assembly code of our Dart program but is not able to copy it in any way.

If this is an expected behavior of tables, I would like to suggest having some kind of "Export as Text" function in all views so we can at least export the information out of DevTools instead of relying on screenshots. :)
